src/connectionManager.ts

```diff
@@ -0,0 +1,84 @@
+// TODO: Reference a library instead for GUID generation?
```

Comment:
Yes, I think we should. I've never had a problem with [this one](https://github.com/uuidjs/uuid).